packages/block-brokers/.aegir.js

```diff
@@ -46,6 +46,27 @@ const options = {
         // "hello"
         res.end(Uint8Array.from([104, 101, 108, 108, 111]))
       })
+
+      goodGateway.all('/ipfs/bafkreig7p6kzwgg4hp3n7wpnnn3kkjmpzxds5rmwhphyueilbzabvyexvq', async (req, res) => {
+        // if 'delay' header is set, delay the response
+        const delay = req.headers['delay']
+        if (delay) {
+          await new Promise((resolve) => setTimeout(resolve, delay))
+          res.writeHead(200, {
+            'content-type': 'application/octet-stream',
+            'content-length': 4
+          })
+          res.end(Uint8Array.from([0, 1, 2, 0]))
+        } else {
+          // no reason to use this in tests without the delay configured.
+          res.writeHead(500, {
```

Comment:
If a required header is missing, maybe use "bad request" instead of "internal server error"?

```suggestion
          res.writeHead(400, {
```